src/index.ts

```diff
@@ -345,12 +345,21 @@ function resolveDevServerUrl(address: AddressInfo, config: ResolvedConfig): DevS
 
     const configHmrHost = typeof config.server.hmr === 'object' ? config.server.hmr.host : null
     const configHost = typeof config.server.host === 'string' ? config.server.host : null
-    const serverAddress = address.family === 'IPv6' ? `[${address.address}]` : address.address
+    const serverAddress = isIpV6(address) ? `[${address.address}]` : address.address
     const host = configHmrHost ?? configHost ?? serverAddress
 
     return `${protocol}://${host}:${address.port}`
 }
 
+function isIpV6(address: AddressInfo): boolean {
```

Comment:
I think `isIpv6` might read a bit better. I think generally initialisms in camel case are treated like they're a word. E.g. `isApi`

Reply:
good point! Updated.